demo/.eslintrc.js

```diff
@@ -5,19 +5,21 @@ module.exports = {
     es6: true,
     node: true,
   },
-  plugins: ["react", "prettier", "react-native"],
+  plugins: ["react", "prettier", "react-native", "jest"],
```

Comment:
@marcelomorgado Hmm, why is this still showing up in the diff? Maybe it stays here until we merge the changes from develop into this branch? I would think it would already recognize that this is no longer a diff relative to develop.

Reply:
Weird, develop branch is updated: https://github.com/tasitlabs/tasit/blob/ec81f57d1ee42b9a49bcea6891a305c20988ea78/demo/.eslintrc.js#L8

Reply:
As an experiment I checked out develop, pulled the latest changes from tasitlabs, fetched your remote, checked out the feature/snapshots feature branch, and did `git merge --no-ff develop`, then pushed that to my fork and did a diff with develop. It disappears after doing that https://github.com/tasitlabs/tasit/compare/develop...pcowgill:feature/snapshots

Reply:
Hmmm

Reply:
Thanks.